src/plots/cartesian/select.js

```diff
@@ -404,18 +404,20 @@ function updateSelectedState(gd, searchTraces, eventData) {
         var len = items.length;
         var item0 = items[0];
         var trace0 = item0.cd[0].trace;
+        var _module = item0._module;
+        var fn = _module.styleOnSelect || _module.style;
```

Comment:
I like `styleOnSelect`, very nice solution!

But please call it something more descriptive than `fn` here - `styleSelection`?

Reply:
sure thing :wink:

Reply:
done in 0e91c6e